Plugins/DeleteUnusedBranches/DeleteUnusedBranchesForm.cs

```diff
@@ -112,7 +118,16 @@ private static IEnumerable<string> GetObsoleteBranchNames(RefreshContext context
             var regex = string.IsNullOrEmpty(context.RegexFilter) ? null : new Regex(context.RegexFilter, options);
             bool regexMustMatch = !context.RegexDoesNotMatch;
 
-            string[] branches = context.Commands.RunGitCmd("branch" + (context.IncludeRemotes ? " -r" : "") + (context.IncludeUnmerged ? "" : " --merged " + context.ReferenceBranch))
+            // { context.IncludeRemotes, "-r" },
+            // { !context.IncludeUnmerged, "--merged" },
```

Comment:
is this needed?

Reply:
No. I was having issues and finally fixed it. Did not remove comment.